src/utilities/index.js

```diff
@@ -60,3 +60,13 @@ export {
 
 // Asserts that a string is a valid GraphQL name
 export { assertValidName } from './assertValidName';
+
+// Helpful functions to compare two GraphQLSchemas and detect breaking changes.
+export {
+  findBreakingChanges,
```

Comment:
Unit tests should be importing from that file directly - not from this package module.

I don't think there's a good reason to only partially check for breaking changes? Perhaps if you want to filter out certain breakages a better way would be to filter based on data in the BreakingChange objects instead of manually assembling these functions yourself.

If there real need in the future then we could export these then - otherwise I'd prefer to keep the surface area of the exported API smaller and preserve the option value to change the implementation of `findRemovedTypes` in the future.